### ISSUE TITLE
Define export fields based on form

### ISSUE THREAD

**Issue by kszymukowicz** [pagemachine/typo3-formlog#131]
hi
I can not find in docs and from the code looks like its not possible to define export fields based on form.
Do you have some nice trick for that?

**Comment by mbrodala** [comment]
What do you mean exactly with that?

Maybe related to https://github.com/pagemachine/typo3-formlog/issues/17

Here you can see that it's not that easy to implement but maybe you have a good idea.

**Comment by kszymukowicz** [comment]
Sorry for late response. I was trying to find the best solution.

Actually I was inspired by the conditions here https://github.com/pagemachine/typo3-formlog/issues/15 and I have made a condition per form identifier like below. This solves my issue. If you want I can make a PR with docs for that solution or maybe you want to solve that possibility differently? For example by defining some way to inject TS per form identifier on TS level. (f.e. something like `module.tx_formlog.settings.list._form-identifier.f1-5685 { }`)

My working solution:
```
[request.getQueryParams()['tx_formlog_web_formloglist']['filters']['identifier']['value'] == 'f1-5685']
    module.tx_formlog {
        settings {
            list {
               // some custom columns settings per form
            }
            export < .list
        }
    }
[global]
```

The idea of https://github.com/pagemachine/typo3-formlog/issues/17 is nice however. I suppose when my PMs will start to make own form they will need that definitely.

**Comment by mbrodala** [comment]
Your workaround may work but it relies on two facts which cannot be taken as given:

1. Log entries of a single form are listed
2. The form definition never changes

By default we the list contains entries of all forms mixed, filtering by a single identifier may open up the path for exporting entries of a single form. But that form definitions may change cannot be avoided; the same form identifier can contain slightly or even completely different fields over time.

Thus this requires a good concept for a general solution which covers the most cases.

**Comment by mbrodala** [comment]
Giving this another thought the suggested TypoScript setup could indeed help to get this achieved.

Currently we have this:

```
module.tx_formlog.settings.list.columns
module.tx_formlog.settings.export.columns
```

Another option could be added to specify dedicated `columns` for each form identifier:

```
module.tx_formlog.settings.list.columns
module.tx_formlog.settings.export.columns

module.tx_formlog.settings.form.<identifier>.list.columns
module.tx_formlog.settings.form.<identifier>.export.columns
```

This could then prefer the 2nd paths if only entries from the form `<identifier>` are to be listed; we can determine this from the fact that the matching filter is applied. Tasks for this:

1. Introduce the new settings path
2. Have the dedicated columns respected in the list
3. Have the dedicated columns respected in the export
4. Add an indicator to the log module if form-specific column settings are used (hopefully reduces confusion about varying display and export)

**Comment by kszymukowicz** [comment]
Yes - that will solve the issue and your TS settings proposal is better than mine.
